styles.css

```
@@ -0,0 +1,283 @@
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+
+body {
+    font-family: 'Roboto Light', sans-serif;
+    font-style: normal;
+}
+
+a {
+    text-decoration: none;
+}
+
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Merriweather', serif;
+    font-style: bold;
+}
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+.header-container {
+    position: relative;
+    background: linear-gradient(90deg, rgba(87, 34, 205, 1) 0%, rgba(72, 69, 228, 1) 100%);
+}
+
+.nav-menu {
+    position: absolute;
+    top: 10px;
+    right: 25px;
+}
+
+.nav-menu ul li {
+    display: inline-block;
+    margin-right: 35px;
+}
+
+.nav-menu a {
+    text-decoration: none;
+    color: #F2F0FF;
+    font-weight: lighter;
+}
+
+
+.header-content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 80vh;
+}
+
+.header-text {
+    font-size: large;
+    color: #F2F0FF;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+
+.button-prop {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.oval-button {
+    text-transform: uppercase;
+    width: 200px;
+    height: 50px;
+    background-color: #ED4385;
+    box-shadow: 4px 4px 4px rgba(0, 0, 0, 0.3);
+    color: #fff;
+    border: none;
+    border-radius: 25px;
+}
+
+.stripe {
+    position: absolute;
+    bottom: 0;
+    left: 50%;
+    transform: translateX(-50%);
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+.intro-content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 80vh;
+}
+
+.intro-text {
+    color: #5722CD;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    white-space: pre-wrap;
+    line-height: 1.5;
+    font-size: larger;
+}
+
+h2 {
+    font-size: 30px;
+}
+
+.contact {
+    color: #0C0C0C;
+}
+
+/* --------------- 4.3 Projects ------------------------- */
+.projects {
+    padding: 15px;
+    background-color: #F3F6F9;
+}
+
+.project-1, .project-3 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.project-2 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+}
+.project-1, .project-2, .project-3 {
+    margin: 40px;
+    padding: 20px;
+}
+
+.project-text {
+    width: 35%;
+    margin: 60px;
+}
+
+.view-project {
+    color: #5722CD;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 600;
+}
+
+.image-wrapper {
+    flex-basis: 800px;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
```

```suggestion
    align-items: center
}
```
Dan blijft de afbeelding in het midden staan als je de pagina smaller maakt